docs/blog/articles/2024-05-15-Kserve-0.13-release.md

```diff
@@ -0,0 +1,123 @@
+
+* `/openai/v1/completions`
+* `/openai/v1/chat/completions`
+* `/openai/v1/models`
+
+These endpoints are useful for generative transformer models, which take in messages and return a model-generated message output. The [chat completions endpoint](https://platform.openai.com/docs/guides/text-generation/chat-completions-api) is designed for easily handling multi-turn conversations, while still being useful for single-turn tasks. The [completions endpoint](https://platform.openai.com/docs/guides/text-generation/completions-api) is now a legacy endpoint that differs with the chat completions endpoint in that the interface for completions is a freeform text string called a `prompt`. Read more about the [chat completions](https://platform.openai.com/docs/api-reference/chat) and [completions](https://platform.openai.com/docs/api-reference/completions) endpoints int the OpenAI API docs. 
+
+This update fosters a standardized approach to transformer model serving, ensuring compatibility with a broader spectrum of models and tools, and enhances the platform's versatility. The API can be directly used with OpenAI's client libraries or third-party tools, like LangChain or LlamaIndex. 
+
+### Future Plan
+* Support other tasks like text embeddings [#3572](https://github.com/kserve/kserve/issues/3572])
+* Support more LLM backend options in the future, such as TensorRT-LLM.
+* Enrich text generation metrics for Throughput(tokens/sec), TTFT(Time to first token) [#3461](https://github.com/kserve/kserve/issues/3461).
+* KEDA integration for token based LLM Autoscaling [#3561](https://github.com/kserve/kserve/issues/3561).
+
+
+## Other Changes
+
+This release also includes several enhancements and changes:
+
+### What's New?
+* Async streaming support for v1 endpoints [#3402](https://github.com/kserve/kserve/issues/3402).
+* Support for .json and .ubj model formats in the XGBoost server image [#3546](https://github.com/kserve/kserve/issues/3546).
+* Enhanced flexibility in KServe by allowing the configuration of multiple domains for an inference service [#2747](https://github.com/kserve/kserve/issues/2747).
+* Enhanced the manager setup to dynamically adapt based on available CRDs, improving operational flexibility and reliability across different deployment environments [#3470](https://github.com/kserve/kserve/issues/3470).
+
+### What's Changed?
+* Remove Seldon Alibi dependencie [#3380](https://github.com/kserve/kserve/issues/3380).
+* Removal of conversion webhook from manifests. [#3344](https://github.com/kserve/kserve/issues/3344).
+
+For complete details on the new features and updates, visit our [official release notes](https://github.com/kserve/kserve/releases/tag/v0.13.0-rc0).
+
+
+## Join the community
+
+- Visit our [Website](https://kserve.github.io/website/) or [GitHub](https://github.com/kserve)
+- Join the Slack ([#kserve](https://kubeflow.slack.com/?redir=%2Farchives%2FCH6E58LNP))
```

Comment:
Let's use this link that uses the CNCF Slack instead: https://github.com/kserve/community?tab=readme-ov-file#questions-and-issues

Reply:
done